src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -981,6 +982,12 @@ private static void PrintLatencies(List<Worker> workers,
       }
     }
 
+    NewOrder.printLatencyStats();
+    Payment.printLatencyStats();
+    OrderStatus.printLatencyStats();
+    Delivery.printLatencyStats();
+    StockLevel.printLatencyStats();
+
```

Comment:
Can we use a flag to enable/disable this printing?

Reply:
These can be disabled in some sense by setting Log4j for `com.oltpbenchmark.benchmarks.tpcc.procedures`

Reply:
It would also be disabled if we disable tracking the metrics.